docs/setup_guide.md

```diff
@@ -0,0 +1,120 @@
+# Setup Guide
+
+This guide will provide the steps how to setup a DAO module with a Safe on Rinkeby. The setup will use Realitio and can be used with Snapshot spaces.
+
+More information how this works can be found on the [Gnosis blog](https://blog.gnosis.pm/ea67eb95c34f).
+
+## Prerequisites
+
+To start the process you need to create a Safe (e.g. via https://app.gnosis-safe.io). This Safe will represent the DAO and hold all the assets (e.g. tokens and collectibles). It is required that you can execute via the Safe to setup the DAO module.
```

Comment:
Shouldn't this be the Rinkeby link? rinkeby.gnosis-safe.io/app/

~It is required that you can execute via the Safe to setup the DAO module.~
A Safe transaction is required to setup the DAO module.